docs/limitations.md

```diff
@@ -1,12 +1,30 @@
 # Current limitations
 
-1. Does not work on Windows. Tested on Mac and Linux only. Tracking issue: https://github.com/prisma/prisma2/issues/4
-2. Processes 1 request at a time, no parallelism during the early preview period. This will be fixed before GA. 
-3. Some edge cases for complex nested mutations don't work properly. 
-4. Limited auto-complete in Typescript projects due to a compiler bug. We raised a PR to fix that [here](https://github.com/microsoft/TypeScript/pull/32100). 
-5. No realtime API/subscriptions. In the future, we will have Prisma events engine (after the GA) that would pave way for such a feature, there is no ETA for this yet. 
-6. Models must have an `@id` attribute and it must take one of these forms:
+During the preview phase of Prisma 2 there are missing features, limited performance and stability issues you should be aware of:
+
+## Functionality limitations
+
+- `Embed` types are not implemented yet ([tracking issue](https://github.com/prisma/lift/issues/43))
```

Comment:
I'd suggest to only use inline code formatting when this is actually referencing some programmatic concept, so here maybe either lowercase `embed` or remove the inline code formatting.

Reply:
Isn't this a type from our schema? https://github.com/prisma/specs/tree/master/schema#embed-block
But yeah, that is still the "concept" referred to here, not the `embed` actually used in the schema, right?

Reply:
Workaround (which also matches the issue):
```suggestion
- Embedded types are not implemented yet ([tracking issue](https://github.com/prisma/lift/issues/43))
```
Correct?

Reply:
> Isn't this a type from our schema? https://github.com/prisma/specs/tree/master/schema#embed-block

Absolutely! My point was to spell it like this type if you're using inline code (which means it should be lowercased). Sorry if that wasn't clear!

Reply:
I wasn't aware of that before actually visiting that link ;)

What do you think of the suggestion?

Reply:
Looks good 👍